src/protocol/prf_sharding/bucket.rs

```diff
@@ -221,7 +239,7 @@ pub mod tests {
                             RecordId::from(0),
                             breakdown_key_share,
                             value_share,
-                            129,
+                            513,
```

Comment:
Up on line 228, you need to find a different GfXBit type for this function.  Otherwise you'll never get past 256 buckets.